vwifi.c

```diff
@@ -71,6 +74,10 @@ static DEFINE_SPINLOCK(vif_list_lock);
 /* SME stands for "station management entity" */
 enum sme_state { SME_DISCONNECTED, SME_CONNECTING, SME_CONNECTED };
 
+/* Each virtual interface contains a wiphy, wvifi_wiphy_counter is responsible
```

Comment:
Tweak the comment style. i.e.,
```c
/* Each virtual interface contains a wiphy, wvifi_wiphy_counter is responsible
 * for recording the number of wiphy in vwifi.
 */
```

Be aware of the position of the ending mark.